### ISSUE TITLE
Error when using channels

### ISSUE THREAD

**Issue by Neamar** [subosito/flutter-action#39]
Hello!

Since the last Flutter update three days ago, the channel feature seems broken :(

Here is an excerpt:

```yml
      - uses: subosito/flutter-action@v1
        with:
          channel: 'beta' # flutter web is not stable yet
      - run: flutter --version
```

Before the Flutter update three days ago:
```
Run flutter --version
Flutter 1.17.0-3.4.pre • channel beta • https://github.com/flutter/flutter.git
```

Now:
```
Run flutter --version
Flutter 1.17.0 • channel stable • https://github.com/flutter/flutter.git
```

I assume the path to download Flutter archive changed?

Thanks for the GithubAction, it's really neat!

(example live URL: https://github.com/Neamar/turn_me_on/runs/656069282?check_suite_focus=true)

**Comment by Neamar** [comment]
(I had a quick look at the code and I believe this is because the file referenced from https://github.com/subosito/flutter-action/blob/794e42ad48b6f61cd681203e13314877b81cb77a/src/installer.ts#L160 `https://storage.googleapis.com/flutter_infra/releases_windows.json` does not exist anymore, but I'm not sure as I never looked into the code before)

**Comment by subosito** [comment]
Whoa, didn't realize that @Neamar. We rely on the information on https://storage.googleapis.com/flutter_infra/releases_windows.json, but now it's blank. Linux and macOS are affected too.

Any updates about the replacement of these manifest would be great, so we can update it and make everything works normal again.

**Comment by janstol** [comment]
@subosito I found it! (https://github.com/flutter/flutter/wiki/Release-process) 

- https://storage.googleapis.com/flutter_infra/releases/releases_linux.json
- https://storage.googleapis.com/flutter_infra/releases/releases_windows.json
- https://storage.googleapis.com/flutter_infra/releases/releases_macos.json

**Comment by Neamar** [comment]
I was going to post the exact same thing :p Cool, should be an easy fix. If needed, I may be able to send a PR but wouldn't know how to test.

**Comment by treeder** [comment]
Is it possible to use a previous version of this action until this gets fixed?  Anyone know how to do that?

**Comment by treeder** [comment]
Or I guess they'd all be broken since where it's getting information changed... ignore... ;)

**Comment by subosito** [comment]
Sorry guys, we already use a correct manifest. I was in a hurry when commenting @Neamar.

The issue is in the information within the manifest itself. As you can see on https://storage.googleapis.com/flutter_infra/releases/releases_linux.json:

```json
  "base_url": "https://storage.googleapis.com/flutter_infra/releases",
  "current_release": {
    "beta": "e6b34c2b5c96bb95325269a29a84e83ed8909b5f",
    "dev": "a849daf2836d653ef80e01f428531b3f787671cb",
    "stable": "e6b34c2b5c96bb95325269a29a84e83ed8909b5f"
  }
```

The current release of `beta` and `stable` channel points to the same release. That's why even though we explicitly set to use `beta` channel, it will getting the `stable` one. Previously they always point to a different release.

**Comment by subosito** [comment]
I've released `v1.3.2` and updated `v1` to [include fixes](https://github.com/subosito/flutter-action/commit/4bb529986fcebdc5b610bc807e5fda03a914e6b3) for this issue. Please retry and let me know the result on your side.

**Comment by Neamar** [comment]
Confirmed fixed! Thanks for this project 😊
